src/main/java/org/terracotta/build/plugins/CopyrightPlugin.java

```diff
@@ -79,6 +75,7 @@ public class CopyrightPlugin implements Plugin<Project> {
 
   @Override
   public void apply(Project project) {
+    Git.getOrInstall(project);
```

Comment:
Why is this needed? Shouldn't this just pickup the service reference if it's there?

Reply:
My understanding of the `@ServiceReference` Javadoc is that it will pick up the shared service found with the given name, not create it. Besides, the javadoc of the annotation explains what happens if the shared service is not found.